addons/Rakugo/lib/load_global_history.gd

```diff
@@ -19,16 +19,16 @@ func invoke() -> bool:
 	else:
 		save_path += ".res"
 
-	r.debug(["load global history from:", save_name])
+	r.debug(["load global history from: %s" %save_name])
```

Comment:
`Rakugo.debug()` use array of strings and join them using space.
So this change don't change any thing.